.stylelintrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I'll leave this comment just as a reminder. This is an example of the config that I am using in my projects:

```js
// stylelint.config.js
module.exports = {
  plugins: ['stylelint-prettier', 'stylelint-no-unsupported-browser-features'],
  ignorePath: ['dist'],
  extends: [
    'stylelint-config-recommended',
    'stylelint-config-sass-guidelines',
    'stylelint-prettier/recommended',
  ],
  rules: {
    'no-invalid-position-at-import-rule': null,
    'scss/selector-no-redundant-nesting-selector': true,
    'scss/at-import-no-partial-leading-underscore': true,
    'scss/at-mixin-pattern': null,
    'scss/function-no-unknown': [
      true,
      {
        ignoreFunctions: ['palette-value', 'get_font_property_value', 'str-replace', 'check-map-exists', 'strip-unit']
      }
    ],
    'function-no-unknown': [
      true,
      {
        ignoreFunctions: ['palette-value', 'get_font_property_value', 'str-replace', 'check-map-exists', 'strip-unit']
      }
    ],
    'property-no-vendor-prefix': true,
    'color-named': 'never',
    'color-no-hex': true,
    'shorthand-property-no-redundant-values': true,
    'declaration-block-no-redundant-longhand-properties': true,
    'max-nesting-depth': [3, { severity: 'warning' }],
    'selector-max-compound-selectors': [3, { severity: 'warning' }],
    'selector-class-pattern': null,
    'property-disallowed-list': ["font-size, letter-spacing, line-height, text-transformation, font-family, font-weight, float"],
    'at-rule-disallowed-list': ["media"],

    'selector-no-qualifying-type': [
      true,
      {
        ignore: ['attribute'],
      },
    ],

    'order/order': [
      'dollar-variables',
      'custom-properties',
      'at-rules',
      'declarations',
      'rules',
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'breakpoint',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'mobile',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'tablet-landscape',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'tablet',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'desktop',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'sidebarpluscontent',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'small-screen',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'medium-screen',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'almost-large-screen',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'large-screen',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'include',
        parameter: 'mainmenuspecial',
        hasBlock: true,
      },
      {
        type: 'at-rule',
        name: 'media',
        hasBlock: true,
      },
    ],
    'order/properties-alphabetical-order': null,

    'declaration-empty-line-before': null,
    'max-empty-lines': 2,
    'selector-pseudo-class-no-unknown': [
      true,
      {
        ignorePseudoClasses: ['local', 'global', 'export'],
      },
    ],
    'property-no-unknown': [
      true,
      {
        ignoreProperties: ['composes', 'compose-with'],
      },
    ],
    'no-descending-specificity': null,
    'comment-whitespace-inside': null,
    'no-eol-whitespace': null,
    'plugin/no-unsupported-browser-features': [
      true,
      {
        ignore: [
          /* IE11 have a large amount of bugs present ( https://caniuse.com/#feat=flexbox ) ::
           * is only partially supported by IE 11 */
          'flexbox',

          /* calc is not working correctly with various use cases ( https://caniuse.com/#search=calc ) ::
           * only partially supported by IE 11, Android Browser 4.4.3-4.4.4 */
          'calc',

          /* don't need cursor in mobile ::
           * is not supported by iOS Safari 8,10.3,11.0-11.2,11.3-11.4,12.0-12.1, Android Browser 4.4.3-4.4.4, UC Browser for Android 11.8, Samsung */
          'css3-cursors',

          /* progressive enhancement ::
           * is only partially supported by Safari 12,11.1, iOS Safari 8,10.3,11.0-11.2,11.3-11.4,12.0-12.1 */
          'css-gradients',

          /* viewport-units are very usefull, just don't use vmax ::
           * is only partially supported by IE 11 (ie11 don't support 'vmax' unit ) */
          'viewport-units',

          /* progressive enhancement ::
           * is not supported by IE 11 and only partially supported by Edge 17,18, Firefox 64,65, Chrome 49,61,63,70,71,72, Safari 12,11.1, Opera 57,58,  plugin/no-unsupported-browser-features iOS Safari 8,10.3,11.0-11.2,11.3-11.4,12.0-12.1, Android Browser 4.4.3-4.4.4, UC Browser for Android 11.8, Samsung Internet 4,7.2-7.4,8.2 */
          'css-appearance',

          /* progressive enhancement ::
           * ie11 not supported */
          'css-filters',
        ],
        severity: 'warning',
      },
    ],
    // design system specific rules
    'declaration-property-value-disallowed-list' : {
      "color": ["/base-background/"], // disallowed bg color variable as text
      "fill": ["/base-text/"], // disallowed bg color variable as fill
      "background-color": ["/base-text/"], // disallowed text variable as background
      "background": ["/base-text/"], // disallowed text variable as background
      "width": ["/spacing/"], // disallowed spacing variable in width
      "height": ["/spacing/"], // disallowed spacing variable in height
      "padding": ["/[1-9]/"], // disallowed number in padding
      "margin": ["/[1-9]/"], // disallowed  number in margin
    },
    'declaration-property-value-allowed-list' : {
      "padding": ["/spacing/", "/0/"], // only allow spacing variable or 0 in padding & margin
      "margin": ["/spacing/", "/0/"],
    },
  },
}
```